### PR TITLE
Document composition principles and CSS guidelines

### PR DESCRIPTION
**Pull request by adamkudrna**
New sections:

- Contribute > Composition
- Contribute > CSS Guidelines

A PR with refactored CSS to match the new guidelines will follow as soon as #301 is merged.

### CODE REVIEW

File: src/docs/contribute/css.mdx [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+- **Classes that are automatically generated by Sass loops should be handled
+  with care.** With loops, it's easy to produce a lot of CSS and negatively
+  impact performance.
+
+## Postprocessing with PostCSS
+
+All styles are automatically prefixed by [Autoprefixer] plugin for [PostCSS]
+according to [Browserslist] configuration stored in `.browserslistrc`.
+
+[Stylelint]: https://stylelint.io
```
Comment (mbohal) [inline src/docs/contribute/css.mdx:237]: These links do not get rendered in final docs.

Also why are they here? I think we should make a hyperlink out of name used in text, but this is just a bunch of more or less related texts. 

It makes it feel like a CSS tutorial and not a library documentation.

**Review by mbohal — APPROVED** [review]

File: src/docs/contribute/css.mdx (hunk 1 above)
Comment (adamkudrna) [reply src/docs/contribute/css.mdx:237]: They do render, it's called [reference links](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links).

Comment (mbohal) [reply src/docs/contribute/css.mdx:237]: I dont see them:
Screenshot from 2021-09-16 10-53-08

Comment (bedrich-schindler) [reply src/docs/contribute/css.mdx:237]: @mbohal It is not supposed to be rendered, see the link. It is just definition so you do not have to have long links in the main texts and you use those predefined instead.

**Review by bedrich-schindler — APPROVED** [review]

File: src/docs/contribute/css.mdx (hunk 1 above)
Comment (mbohal) [reply src/docs/contribute/css.mdx:237]: Everyday I learn something new…

**Pull request merged**